style.css

```diff
@@ -601,3 +794,477 @@ button:hover {
+    margin-right: 612px;
+    margin-top: 170px;
+  }
+
+  .button1 {
+    width: 149px;
+    margin-left: 48px;
+    margin-top: 40px;
+  }
+
+  .push1 {
+    padding-left: 0;
+    width: 203px;
+    margin-left: 48px;
+  }
+
+  .third_work_container {
+    grid-template-columns: 1fr 1fr;
+    width: 1165px;
+    background: #fff;
+    margin-left: 142px;
+    height: 496px;
+  }
+
+  .third_wk {
+    position: absolute;
+    left: 118px;
+    top: 2170px;
+    width: 544px;
+    height: 448px;
+  }
+
+  .workthree {
+    position: absolute;
+    margin-left: 614px;
+    font-size: 40px;
+    width: 496px;
+    line-height: 52px;
+    margin-top: 48px;
+  }
+
+  .sec_h3 {
+    position: absolute;
+    margin-left: 619px;
+    top: 2270px;
+    width: 298px;
+  }
+
+  .p3 {
+    display: flex;
+    align-items: flex-start;
+    font-size: 16px;
+    padding: 5px;
+    height: 70px;
+    width: 496px;
+    margin-left: 614px;
+    margin-right: 46px;
+    margin-top: 170px;
+  }
+
+  .push2 {
+    padding-left: 0;
+    width: 203px;
+    margin-left: 614px;
+  }
+
+  .button3 {
+    width: 149px;
+    margin-left: 614px;
+    margin-top: 40px;
+  }
+
+  .forth_work_container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 1165px;
+    margin-left: 142px;
+    height: 496px;
+  }
+
+  .forth_wk {
+    position: absolute;
+    left: 690px;
+    top: 2775px;
+    width: 544px;
+    height: 448px;
+  }
+
+  .workfour {
+    position: absolute;
+    margin-top: 48px;
+    margin-left: 48px;
+    height: 46px;
+    width: 511px;
+    line-height: 52px;
+    font-size: 40px;
+  }
+
+  .sec_h4 {
+    position: absolute;
+    top: 2865px;
+    margin-left: 48px;
+    width: 320px;
+  }
+
+  .p4 {
+    display: flex;
+    align-items: flex-start;
+    font-size: 16px;
+    padding: 5px;
+    height: 70px;
+    width: 496px;
+    margin-left: 48px;
+    margin-right: 612px;
+    margin-top: 170px;
+  }
+
+  .about_my {
+    width: 251px;
+    height: 114px;
+    margin-top: 142px;
+    margin-left: 142px;
+  }
+
+  .sec_abt {
+    margin-left: 142px;
+  }
+
+  .contact_section2 {
+    margin-left: 147px;
+    margin-top: 0;
+  }
+
+  .abt_icn {
+    display: flex;
+    justify-content: space-evenly;
+    margin-left: 147px;
+    width: 200px;
+    margin-top: 0;
+  }
+
+  .button5 {
+    margin-left: 147px;
+    margin-top: 24px;
+  }
+
+  .fr_sk {
+    position: absolute;
+    width: 684px;
+    height: 360px;
+    top: 3365px;
+    left: 612px;
+  }
+
+  .dropim {
+    width: 582px;
+    position: absolute;
+    left: 24px;
+    height: 24px;
+    color: #344563;
+  }
+
+  .lasjss {
+    width: 122px;
+    position: absolute;
+    top: 84px;
+    left: 24px;
+    height: 120px;
+  }
+
+  .lasht {
+    width: 122px;
+    left: 162px;
+    height: 120px;
+    position: absolute;
+    top: 84px;
+  }
+
+  .lascs {
+    position: absolute;
+    left: 297px;
+    top: 84px;
+    width: 122px;
+    height: 120px;
+  }
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly improve your styling on each skill to match the Figma design as much as you can below:

*your design*
![skillss-s](https://user-images.githubusercontent.com/59397159/172935961-b48faeb0-c5a2-4995-b656-749a969c175a.png)


*figma*
![skills](https://user-images.githubusercontent.com/59397159/172935967-76c136d6-65f0-4c3c-8388-4b33988d3360.png)